tick/inference/src/hawkes_sumgaussians.h

```diff
@@ -0,0 +1,99 @@
+#ifndef MLPP_INFERENCE_SRC_HAWKES_SUMGAUSSIANS_H_
+#define MLPP_INFERENCE_SRC_HAWKES_SUMGAUSSIANS_H_
+
+#include "base.h"
+#include "base/hawkes_list.h"
+
+/**
+ * \class HawkesSumGaussians
+ * \brief Class for implementation of the algorithm described in the paper
+ * `Learning Granger Causality for Hawkes Processes`
+ *  by Xu, Farajtabar, and Zha (2016) in ICML
+ */
+class HawkesSumGaussians : public ModelHawkesList {
+  //! @brief Number of gaussian functions to approximate each kernel
+  int n_gaussians;
```

Comment:
type should be either int or ulong but your switching too often :), same for em_max_iter.